servers/republik/modules/crowdfundings/lib/translations.json

```
@@ -479,8 +479,8 @@
       "value": "Möchten Sie Ihre Abo verlängern oder die Republik verlassen? Entscheiden Sie sich bis {endDate}"
```

```suggestion
      "value": "Möchten Sie Ihr Abo verlängern oder die Republik verlassen? Entscheiden Sie sich bis {endDate}"
```
typo (fix in gsheet)